src/main/resources/db/data/V016__Insert_personal_data.sql

```diff
@@ -195,10 +195,10 @@ VALUES ( 'Petra', 'Trojković', '218570d0e4590b3455ee2c47606fed85ebc5370aa9b2e13
 --pin:54887728537 phin:709016341
 INSERT INTO personal_data ( first_name, last_name, pin, phin, date_of_birth)
 VALUES ( 'Matija', 'Fabić', '077f84994ec1d55d5d86f5237e779f3ea791402ec27cb5ea09d52109d66df05b',
-         '9775781a98554db8d7731166653474cafdcaa5c8ad2cf51289af7c4cbaaef0eb', '1977-10-10');
```

Comment:
Remove changes from this schema. Schema is fixed already and should not be changed.